examples/replset-docker/README.md

```diff
@@ -0,0 +1,81 @@
+# MongoDB Replication Example Using a Docker
+
+This [MongoDB replication](https://docs.mongodb.com/manual/replication/) example
+uses a [Docker engine](http://docker.com) to run replica set members. This platform
+do not provide any deployment automation and it is used mainly for developing and testing.
```

Comment:
> Why would one use Docker and not use at least Docker Compose?!

@rhcarvalho Because docker is IMPOV "standard" for developers, Docker Compose requires another program to install and things to read about.
It definitely would be good to mention Docker Compose and ideally prepare configuration for it to be able to easily start replication.

I would like to prepare same way for deploying replication as other database containers offers (mariadb, mysql, postgresql).